test/runtests.jl

```diff
@@ -1,30 +1,35 @@
-# TODO: more tests!
-
 using ChainRules, Test, FDM, LinearAlgebra, LinearAlgebra.BLAS, Random, Statistics
-using ChainRules: extern, accumulate, accumulate!, store!, @scalar_rule,
+using Base.Broadcast: broadcastable
+import LinearAlgebra: dot
```

Comment:
Do we not also need to include an `if` check here for which version of Julia we are running?

> https://github.com/JuliaDiff/ChainRules.jl/blob/master/src/ChainRules.jl#L9

Reply:
I don't think so. As we did not `using Blas`